umbrella/Cargo.toml

```diff
@@ -290,7 +290,6 @@ runtime-benchmarks = [
 	"pallet-membership?/runtime-benchmarks",
 	"pallet-message-queue?/runtime-benchmarks",
 	"pallet-migrations?/runtime-benchmarks",
-	"pallet-mixnet?/runtime-benchmarks",
```

Comment:
This should not be removed from what I can say -- This pallet will still have `runtime-benchmarks` feature.

Reply:
There is no `bechmarking.rs` for `pallet-mixnet`, hence I remove this. It builds without any issue. Could you please explain a bit as to why you think that this should be still included?

Reply:
Added it back for now. Will understand the context for this offline.

Reply:
Cool, thanks. Removed it.